feature-home/src/main/java/cm/aptoide/pt/feature_home/presentation/BundlesViewModel.kt

```diff
@@ -14,10 +14,10 @@ import timber.log.Timber
 import java.io.IOException
 import javax.inject.Inject
 
-@HiltViewModel
-class BundlesViewModel @Inject constructor(
+abstract class BundlesViewModel(
```

Comment:
There is no need to make it abstract.
Better to avoid inheritance when possible :-)